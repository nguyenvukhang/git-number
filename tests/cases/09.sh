# not from git root

init 3
# create and commit ./src
mkdir src
touch src/.gitkeep
_gitnu add src
_gitnu commit -m "src_dir"

cd src
touch emerald sapphire ruby
_gitnu status
_gitnu add 3-5
log gitnu status
