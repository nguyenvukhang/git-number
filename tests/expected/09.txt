On branch main
Changes to be committed:
1	[32mnew file:   ../file_0003[m
2	[32mnew file:   emerald[m
3	[32mnew file:   ruby[m

Untracked files:
4	[31m../file_0001[m
5	[31m../file_0002[m
6	[31msapphire[m

